### PR TITLE
feat: compress urange

### PR DESCRIPTION
**Pull request by alexander-akait**
**Description:**

Port - https://github.com/cssnano/cssnano/tree/master/packages/postcss-normalize-unicode + drop leading `0`

**BREAKING CHANGE:**

No

**Related issue (if exists):**

No

### CODE REVIEW

File: crates/swc_css_minifier/src/compress/urange.rs [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+}
+
+struct CompressUrange {}
+
+impl CompressUrange {
+    fn remove_leading_zeros(&mut self, value: &str) -> String {
+        let mut result = String::new();
+        let mut is_leading = true;
+
+        for char in value.chars() {
```
Comment (kdy1) [inline crates/swc_css_minifier/src/compress/urange.rs:15]: Please use different name than `char`, which is a builtin type.
This distracts review process.

**Comment by alexander-akait** [comment]
@kdy1 Ready for review

File: crates/swc_css_minifier/src/compress/urange.rs (hunk 1 above)
Comment (kdy1) [reply crates/swc_css_minifier/src/compress/urange.rs:15]: This still applies

**Comment by alexander-akait** [comment]
@kdy1 missed that :smile: Fixed

**Comment by alexander-akait** [comment]
@kdy1 Rebased too

**Review by kdy1 — CHANGES_REQUESTED** [review]
Mostly LGTM.

I found one code style issue.

**Review by kdy1 — APPROVED** [review]
LGTM. Thank you!

---

swc-bump:
 - swc_css_minifier

**Pull request merged**